src/com/google/javascript/jscomp/CommandLineRunner.java

```diff
@@ -232,6 +232,11 @@
         + "or property_renaming_report")
     private boolean createNameMapFiles = false;
 
+    @Option(name = "--source_map_include_sources_content",
+        handler = BooleanOptionHandler.class,
+        usage = "Includes sources content into source map")
```

Comment:
Should also add that this greatly increases the size of the source map but offers greater portability.